logs/analyze.go

```diff
@@ -2266,8 +2266,7 @@ func AnalyzeBackendLogLines(logLines []state.LogLine) (logLinesOut []state.LogLi
 		logLinesOut = append(logLinesOut, logLine)
 	}
 
-	// Ensure no other part of the system accidentally sends log line contents, as
-	// they should be considered opaque from here on
+	// Remove log line content. Note that ReplaceSecrets adds it back after secrets have been removed.
 	for idx := range logLinesOut {
 		logLinesOut[idx].Content = ""
```

Comment:
A future PR might remove the tempfile being written since I think it's not needed now. At that point we wouldn't need to zero-out the log text here. Though that would also require refactoring the log filter code so the byte offsets are relative to that string, instead of being relative to the entire file.